content/en/docs/v3.5/learning/api_guarantees.md

```diff
@@ -4,7 +4,10 @@ weight: 2750
 description: KV API guarantees made by etcd
 ---
 
-etcd is a consistent and durable key value store with [mini-transaction][txn] support. The key value store is exposed through the KV APIs. etcd tries to ensure the strongest consistency and durability guarantees for a distributed system. This specification enumerates the KV API guarantees made by etcd.
+etcd is a consistent and durable key value store with [mini-transaction][txn]
+support. The key value store is exposed through the KV APIs. etcd tries to
+ensure the strongest consistency and durability guarantees for a distributed
+system. This specification enumerates the KV API guarantees made by etcd.
```

Comment:
Want to double check - is the splitting of paragraph onto separate lines all through this file intentional?  It can create problems when viewing the paragraph on different size screens, also doesn't align with main etcd documentation style.

Reply:
This is markdown. Splitting lines in markdown does not impact the format of outputted file as long there is no empty line.

Reply:
Fair point, though we are reliant on our website docs correctly rendering/interpreting and some hugo sites do interesting things with templated markdown rendering overrides. I haven't looked in close detail about how etcd site is templated/rendered yet.

In general my preference would be to write paragraphs as paragraphs instead of doing seperate lines and relying on Markdown features that may not be intuitive and also differs in style to how our other docs are generally written from the ones I have looked at so far.

I don't think this feedback should block merge, it's just a suggestion.

Reply:
I'm not sure I understand. I split the lines so it's easier to edit in markdown. I expect no changes on rendered webpage.

> In general my preference would be to write paragraphs as paragraphs instead of doing seperate lines

I think that only works when writing in rich text editor (Microsoft Word, Google docs etc) not in markdown.

Reply:
I'll approve the line splits, but I'm not convinced I want to make them part of the style guide for Etcd docs.  We should discuss whether to do that as a separate issue.